app/bundles/ApiBundle/Controller/FetchCommonApiController.php

```diff
@@ -591,7 +588,7 @@ protected function prepareEntityResultsToArray($results, $callback = null): arra
             $totalCount = count($results);
         }
 
-        $entities = $this->entityResultHelper->getArray($results, $callback);
+        $entities = (new EntityResultHelper())->getArray($results, $callback);
```

Comment:
I would prefer to use the dependency injection just for this controller. This would improve the testability of the code.

Reply:
This controller is extended by many. I'll close this as it may cause more harm than this cleanup is worth.